tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "resolveJsonModule": true,
     "declaration": true,
     "declarationDir": "lib/types",
-    "lib": ["es2017", "ESNext.AsyncIterable", "dom", "webworker"]
+    "lib": ["es2017", "ESNext.AsyncIterable", "dom"]
```

Comment:
So removing the `webworker` lib `tsc --no-emit` now validates the types? 

I wonder what types we lose by removing that lib. I'd suspect it has something to do with web workers 😛

Reply:
My guess is that we don't use any of the webworker specific types, although I'm not 100% sure of that.
Do you think this is better than using the `skipLibCheck` option? 

> So removing the webworker lib tsc --no-emit now validates the types
Correct. That command was also validating our types before this change, but it was throwing errors due to duplicate types in `dom` and `webworker`. See https://github.com/microsoft/TypeScript/issues/20595 for more info.

In that issue, someone mentioned to add the types of `webworker` manually.  I tried to do that, but the issue was already resolved by removing the `webworker` lib.